packages/autorest.typescript/test/smoke/generated/agrifood-data-plane/src/responses.ts

```diff
@@ -169,8 +167,6 @@ export interface AttachmentsDeleteDefaultResponse extends HttpResponse {
 /** Downloads and returns attachment as response for the given input filePath. */
 export interface AttachmentsDownload200Response extends HttpResponse {
   status: "200";
-  /** Value may contain any sequence of octets */
-  body: Uint8Array;
```

Comment:
Why we remove this? Not sure it's possible to add UT.

Reply:
it's deleted by mistake, added back.